test/reviews.spec.js

```diff
@@ -0,0 +1,160 @@
+import request from "supertest"; 
+import runApp from "../src/server.js";
+import { getMovie } from "../src/api.js";
+
+//Mocks
+const mockReview = {
+    author: "John Doe",
+    comment: "This is a test comment",
+    rating: 1,
+};
+
+async function postReview(review, verified = false) { //Posting review to CMS API is not necessary in these tests, so it only returns true
+    return true;
+};
+
+const app = runApp({ //Inject required api functions
+    getMovie,
```

Comment:
I would mock this too, so that your test suite doesn't stop working because something is changed on the CMS (e.g. the movie is retired from circulation and removed from the database).